packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -684,6 +685,17 @@ const ruleset: any = {
     /// ARM rules without an RPC code
     ///
 
+    LatestVersionOfCommonTypesMustBeUsed: {
```

Comment:
>LatestVersionOfCommonTypesMustBeUsed [](http://example.com/codeflow?start=4&length=36)

When this gets exercised in staging, lets verify that the rule gets enforced only for the new changes being put out and not for all references

Reply:
Made an ADO item for this.